tasks/apm-host-injection.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Check if platform is supported
```

Comment:
These task names get printed to users, so let's be more descriptive. I'd suggest something like `Check if APM injection is supported on this host OS`